xwiki-platform-core/xwiki-platform-image/xwiki-platform-image-lightbox/xwiki-platform-image-lightbox-webjar/src/main/webjar/less/lightbox.less

```diff
@@ -25,8 +26,9 @@
   }
 
   .lightboxDescription {
-    background-color: rgba(0, 0, 0, 0.6);
-    color: #FFFFFF;
+    background-color: @panel-bg;
+    opacity: 0.6;
```

Comment:
Isn't the opacity dangerous in terms of contrast?

Reply:
It was already there in the previously hard coded color. I think this is too much of a change to remove it and there is a genuine point to opacity here (we want to still see content behind).

I insisted to remove it on some other PRs, because transparency was not necessary but just a mean to change the color, and we had easy alternative with full semantically appropriate colors from the theme.